app/views/runs/show.html.erb

```diff
@@ -145,7 +145,7 @@
     {
       list_btn_ids.push(value.id); //returns array of unique id's for the action status buttons on the page
     });
-    return(list_btn_ids);
+    return(list_btn_ids.sort());
```

Comment:
I'm not sure this will work as desired:
```
> ["123","234","345","12","22","9"].sort();
< ["12", "123", "22", "234", "345", "9"]
> [123,234,345,12,22,9].sort();
< [12, 123, 22, 234, 345, 9]
```

See [some suggestions here](http://stackoverflow.com/questions/1063007/how-to-sort-an-array-of-integers-correctly), but note that I don't think any handle for NaN. Maybe jQuery can take this from us?